app/models/user.rb

```diff
@@ -86,6 +86,14 @@ def active_profile
     profiles.find(&:active?)
   end
 
+  # This user's most recently activated profile that has also been deactivated
```

Comment:
Could we put this on the `UserDecorator` class instead? We've been trying to keep logical extensions to the `User` model there.